### PR TITLE
feat: Add meta description for snapshot homepage

### LINKED ISSUE #1395: Page description
We should add a page description, currently snapshot appears like this on google 
image

### PR DESCRIPTION
**Pull request by samuveth**
Fixes #1395

## Changes

Add description meta tag for homepage

### CODE REVIEW

**Comment by zzuziak** [comment]
Shall we also add some meta tags with images for URL embedding?

File: index.html [hunk 1]
```diff
@@ -8,6 +8,10 @@
       content="width=device-width, initial-scale=1.0, minimum-scale=1.0, user-scalable=no"
     />
     <title>Snapshot</title>
+    <meta
+      name="description"
+      content="Snapshot is an off-chain voting platform that allows DAOs, DeFi protocols, or NFT communities to vote easily and without gas fees."
```
Comment (zzuziak) [inline index.html:13]: Let's try to make it more googlable:

```suggestion
      content="Snapshot is an off-chain voting platform that allows DAOs, DeFi protocols, or NFT communities to participate in the decentralized governance easily and without gas fees."
```

**Review by zzuziak — COMMENTED** [review]
Amazing, thank you @samuveth :heart: Small suggestion, might not be best but let's try to use diverse vocabulary to make it easier for the search engines :pray:

**Comment by samuveth** [comment]
@zzuziak yes, we just need to get the images then I can add them for `:og` and `:twitter` tags

"Reconciling the guidelines for the image is simple: follow Facebook’s recommendation of a minimum dimension of 1200×630 pixels and an aspect ratio of 1.91:1, but adhere to Twitter’s file size requirement of less than 1MB."

**Comment by zzuziak** [comment]
:rocket:

**Pull request merged**